### PR TITLE
Bank cap rpc

### LINKED ISSUE #4750: Expose RPC API for Bank::capitalization()
Blockexplorers want to know the total supply.  We can get it from here:
https://github.com/solana-labs/solana/blob/7ea522e8511aa85767e8dbd000cc6dd54167bf07/runtime/src/bank.rs#L1234
and surface through an RPC API

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem
Block explorers want access to the current total supply of Sols

#### Summary of Changes
Expose `bank::capitalization()` via `getSolTotalSupply` RPC method

Fixes #4750

### CODE REVIEW

File: core/src/rpc.rs [hunk 1]
```diff
@@ -228,6 +232,9 @@ pub trait RpcSol {
     #[rpc(meta, name = "getTransactionCount")]
     fn get_transaction_count(&self, _: Self::Metadata) -> Result<u64>;
 
+    #[rpc(meta, name = "getSolTotalSupply")]
```
Comment (mvines) [inline core/src/rpc.rs]: nit: let's drop Sol, so `getTotalSupply`.  We're returning Lamports here not Sol

File: core/src/rpc.rs [hunk 2]
```diff
@@ -649,6 +665,34 @@ mod tests {
...
+        /* BOOTSTRAP_LEADER_LAMPORTS get minted for each
+         * bootstrap leader account:
+         * 1) Node account
+         * 2) Voting account
+         * 3) Staking account
+         *
+         * XXX: LEADER_ACCOUNT_COUNT needs to be updated if the number of accounts required
+         * for the bootstrap leader changes.
+         */
+        const LEADER_ACCOUNT_COUNT: u64 = 1 + 1 + 1;
```
Comment (t-nelson) [inline core/src/rpc.rs]: Got a little weird here. All ears as to a better solution!

**Comment by mvines** [comment]
Update https://github.com/solana-labs/solana/blob/master/book/src/jsonrpc-api.md as well please

File: core/src/rpc.rs (hunk 2 above)
Comment (mvines) [reply core/src/rpc.rs]: 🙈 good enough

**Comment by t-nelson** [comment]
:pray:

I'll look into the web3.js side on Monday (or earlier if this rain keeps up...)

**Pull request merged**